src/ui/Controllers/OrganisationController.cs

```diff
@@ -80,25 +65,21 @@ public async Task<ViewResult> About(string ucasCode)
         [Route("{ucasCode}/about")]
         public async Task<ActionResult> AboutPost(string ucasCode, OrganisationViewModel model)
         {
-            var enrichmentModel = (await _manageApi.GetEnrichmentOrganisation(ucasCode)).EnrichmentModel;
+            var ucasInstitutionEnrichmentGetModel = await _manageApi.GetEnrichmentOrganisation(ucasCode);
```

Comment:
The get model will be null on the first enrichment save for an organisation and will blow up on line 77

Reply:
This is dealt with in _manageApi